pullrequest.rst

```diff
@@ -393,6 +393,22 @@ code and leave comments in the pull request or issue tracker.
    However, please be aware that if you are recommending a pull request as
    'merge-ready', you should always make sure the entire test suite passes.
 
+Leaving a Pull Request Review on GitHub
+---------------------------------------
+
+When you review a pull request, you should provide additional details and context
+of your review process. 
+
+Instead of simply "approving" the pull request, leave comments, for example:
+
+1. Whether you have tested the PR
+
+2. Whether the PR fixes the linked bug/issue
+
+3. If you requested changes to the pull request, provide suggestions on how it
+   can be improved.
+
+4. Leave comments on what's "good" on the pull request, not just the "bad".
```

Comment:
``` suggestion
4. Comment on what is "good" about the pull request, not just the "bad".
```
(While this can be overdone,) I think this is important. But I expect most people to not see why, or why they should bother.  So I think something more is needed.

When someone dismisses one of my IDLE improvements as 'useless', it is disheartening.  It shows that likely no effort was made to understand how it might benefit some users.  It also makes it hard to extract anything useful from the communication to make a better improvement.

My suggested addition: `Doing so will make it easier for the PR author to find the good in your comments.`